cicecore/cicedynB/dynamics/ice_dyn_evp.F90

```diff
@@ -1688,55 +1658,17 @@ subroutine stressC_T  (nx_block, ny_block , &
          i = indxti(ij)
          j = indxtj(ij)
 
-         !-----------------------------------------------------------------
-         ! strain rates at T point
-         ! NOTE these are actually strain rates * area  (m^2/s)
-         !-----------------------------------------------------------------
-
-! tcraig bit-for-bit here, not below
-#if (1 == 0) 
-         call strain_rates_T (nx_block,   ny_block,   &
-                              i,          j,          &
-                              uvelE,      vvelE,      &
-                              uvelN,      vvelN,      &
-                              dxN,        dyE,        &
-                              dxT,        dyT,        &
-                              divT,       tensionT,   &
-                              shearT,     DeltaT      )
-
-#else
-#if (1 == 0)
-         call strain_rates_T (nx_block,   ny_block,   &
-                              i,          j,          &
-                              uvelE,      vvelE,      &
-                              uvelN,      vvelN,      &
-                              dxN,        dyE,        &
-                              dxT,        dyT,        &
-                              divT = divT,            &
-                              tensionT = tensionT     )
-         call calc_shearT_DeltaT (shrU(i,j),     shrU(i,j-1), &
-                                  shrU(i-1,j-1), shrU(i-1,j), &
-                                  divT(i,j),     tensionT(i,j),    &
-                                  shearT,        DeltaT    )
-#else
-         !-----------------------------------------------------------------
-         ! strain rates
-         ! NOTE these are actually strain rates * area  (m^2/s)
-         !-----------------------------------------------------------------
-
          shearTsqr = (shrU(i,j)**2 + shrU(i,j-1)**2 + shrU(i-1,j-1)**2 + shrU(i-1,j)**2)/4d0
```

Comment:
This should take into account the uarea.

Reply:
I have noted these potential changes in the google doc. Overall the PR is very nice. I would approve it and fix later these details.